dev/devicelab/bin/tasks/flutter_engine_group_performance.dart

```diff
@@ -20,13 +20,7 @@ Future<void> _withApkInstall(
   final DeviceDiscovery devices = DeviceDiscovery();
   final AndroidDevice device = await devices.workingDevice as AndroidDevice;
   await device.unlock();
-  try {
-    // Force proper cleanup before trying to install app. If uninstall fails,
-    // we log exception and proceed with running the test.
-    await device.adb(<String>['uninstall', bundleName]);
-  } on Exception catch (error) {
-    print('adb uninstall failed with exception: $error. Will proceed with test run.');
-  }
+  await device.adb(<String>['uninstall', bundleName], canFail: true);
```

Comment:
Lgtm w/ question

Does canFail:true still log the uninstall failure?

Reply:
I had the same thought and decided this was still net better.

Reply:
No. _execute just returns the code. I can test for it and log if you want.

```dart
  if (exitCode != 0 && !canFail) {
    fail('Executable "$executable" failed with exit code $exitCode.');
  }
```

Reply:
... is what I would say if adb returned the exit code, but it returns stdout.

adb uninstall in the LOGs will print an error:

> [2024-09-17 11:02:18.801185] [STDOUT] stdout: Failure [DELETE_FAILED_INTERNAL_ERROR]